### PR TITLE
Add missing dependency install instructions for Linux cross-building

### PR DESCRIPTION
**Pull request by carlossanlop**

### CODE REVIEW

File: docs/workflow/building/coreclr/cross-building.md [hunk 1]
```diff
@@ -57,6 +57,12 @@ This section will go over cross-compiling on Linux. Currently, Linux allows you
 
 Before you can attempt to do any Linux cross-building, you will need to generate the _ROOTFS_ corresponding to the platform you want to target. The script located in `eng/common/cross/build-rootfs.sh` is in charge of effectuating this task. Note that this script must be run with `sudo`, as it needs to make some symlinks to the system that would not be allowed otherwise.
 
+Before running the script, make sure you have the dependency installed:
+
+```bash
+sudo apt install debootstrap
```
Comment (carlossanlop) [inline docs/workflow/building/coreclr/cross-building.md:63]:
```suggestion
Before running the script, make sure you have the dependencies installed:

```bash
sudo apt install debootstrap gcc clang llvm
```

Comment (agocke) [reply docs/workflow/building/coreclr/cross-building.md:63]: What do we need from gcc here?

Comment (carlossanlop) [reply docs/workflow/building/coreclr/cross-building.md:63]: I got the error saying that there was no gcc or clang installed. But maybe we only need clang?

Comment (jkoritzinsky) [reply docs/workflow/building/coreclr/cross-building.md:63]: You may need gcc for building the rootfs, but I'm not sure.

Comment (akoeplinger) [reply docs/workflow/building/coreclr/cross-building.md:63]: where did you get that message, during creating the rootfs or when building the runtime with it? for the latter we should only need one compiler.

Comment (carlossanlop) [reply docs/workflow/building/coreclr/cross-building.md:63]: @akoeplinger I got this message when building the runtime, yes.

Maybe I should close this PR. Looking at the rest of our documentation, I noticed that the main "Requirements" page for Linux mentions that for cross building I need to install debootstrap and a few other packages that I was missing:

https://github.com/dotnet/runtime/blob/main/docs/workflow/requirements/linux-requirements.md#additional-requirements-for-cross-building

**Pull request closed without merging**